phaseshifts/model.py

```
@@ -866,20 +832,14 @@ def gen_atomic(self, **kwargs):
         else:  # assume using atoms from model
-            files = [
-                os.path.join(input_dir, "at_" + atom.element.symbol + ".i")
-                for atom in self.atoms
-            ]
+            files = [os.path.join(input_dir, "at_" + atom.element.symbol + ".i") for atom in self.atoms]
 
         # generate atomic.i input file by appending multiple at.i files
         with open(output_file, "w") as f:
             # loop through each atomic charge density file in list
             for input_file in files:
                 if not os.path.isfile(str(input_file)) or input_file == None:
```

Ruff (0.14.10)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)